application/src/main/java/org/togetherjava/tjbot/commands/help/BookmarksCleanupRoutine.java

```diff
@@ -0,0 +1,35 @@
+package org.togetherjava.tjbot.commands.help;
+
+import net.dv8tion.jda.api.JDA;
+
+import org.togetherjava.tjbot.commands.Routine;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Tells the bookmarks system to clean bookmarks scheduled for removal.
+ */
+public final class BookmarksCleanupRoutine implements Routine {
```

Comment:
similar to my other suggestion, id rename this maybe to `DeleteExpiredBookmarksRoutine`

Reply:
Changed